src/pages/StatisticsChart/index.tsx

```diff
@@ -169,6 +175,15 @@ export default ({ dispatch }: React.PropsWithoutRef<StateWithDispatch>) => {
                 },
               ]}
             />
+            <Axis
+              name="blockNumber"
+              title={!isMobile()}
+              label={{
+                formatter: (text: string) => {
+                  return handleAxios(new BigNumber(text))
```

Comment:
Is `handleAxios` a typo?

Reply:
Yeah. Updated https://github.com/nervosnetwork/ckb-explorer-frontend/pull/412/commits/dffc80469be8f591a4a05d4501810cc2d1ac6c2e